frontend/src/metabase/collections/components/Header/Header.jsx

```diff
@@ -4,75 +4,103 @@ import { Flex } from "grid-styled";
 import { t } from "ttag";
 
 import * as Urls from "metabase/lib/urls";
-import { color } from "metabase/lib/colors";
 import Icon, { IconWrapper } from "metabase/components/Icon";
 import Link from "metabase/components/Link";
 import PageHeading from "metabase/components/type/PageHeading";
 import Tooltip from "metabase/components/Tooltip";
 import CollectionEditMenu from "metabase/collections/components/CollectionEditMenu";
 
-import { ToggleMobileSidebarIcon } from "./Header.styled";
+import {
+  DescriptionTooltipIcon,
+  ToggleMobileSidebarIcon,
+} from "./Header.styled";
 
-export default function Header({
-  collection,
-  isAdmin,
-  isRoot,
-  isPersonalCollectionChild,
-  collectionId,
+function Title({
+  collection: { description, name },
   handleToggleMobileSidebar,
 }) {
   return (
-    <Flex align="center" py={3}>
-      <Flex align="center">
-        <PageHeading className="text-wrap">
-          <ToggleMobileSidebarIcon onClick={handleToggleMobileSidebar} />
-          {collection.name}
-        </PageHeading>
-        {collection.description && (
-          <Tooltip tooltip={collection.description}>
-            <Icon
-              name="info"
-              ml={1}
-              mt="4px"
-              color={color("bg-dark")}
-              hover={{ color: color("brand") }}
-            />
-          </Tooltip>
-        )}
-      </Flex>
+    <Flex align="center">
+      <PageHeading className="text-wrap">
+        <ToggleMobileSidebarIcon onClick={handleToggleMobileSidebar} />
+        {name}
+      </PageHeading>
+
+      {description && (
+        <Tooltip tooltip={description}>
+          <DescriptionTooltipIcon />
+        </Tooltip>
+      )}
+    </Flex>
+  );
+}
+
+function PermissionsLink({ collection, isAdmin, isPersonalCollectionChild }) {
+  const shouldRender =
+    isAdmin && !collection.personal_owner_id && !isPersonalCollectionChild;
```

Comment:
I guess we should now prefer using [`isPersonalCollection` utility](https://github.com/metabase/metabase/blob/4452329a604f372d9fd6451e9db7327e86ed11b3/frontend/src/metabase/collections/utils.js#L9) instead of manually checking `personal_owner_id`

Also, what do you think about giving these `shouldRender` variables different names that would describe _why_ the component should be rendered or not? For example:

**PermissionLink**
```js
const isPersonal = isPersonalCollection(collection) && !isPersonalCollectionChild
const canChangePermissions = isAdmin && !isPersonal
```

**EditMenu**
```
const hasWritePermission = collection && collection.can_write
const canEditCollection = hasWritePermission && !isPersonalCollection(collection)
```

**CreateCollectionLink**
```js
const hasWritePermission = collection && collection.can_write
```

The `can_write` and personal checks are used in a few places, so `Menu` can calculate those and pass as props, example:

```jsx
function Menu(props) {
   const { collection } = props;
   const isPersonal = isPersonalCollection(collection);
   const hasWritePermission = collection && collection.can_write;
   return (
      <Flex ml="auto">
         <PermissionsLink {...props} isPersonal={isPersonal} hasWritePermission={hasWritePermission} />
         <EditMenu {...props} isPersonal={isPersonal} hasWritePermission={hasWritePermission} />
         <CreateCollectionLink {...props} isPersonal={isPersonal} hasWritePermission={hasWritePermission} />
      </Flex>
   )
}
```

Reply:
@kulyk Changed, almost exactly to what you suggested.

By relying on the imported `isPersonalCollection` we now are not checking `isPersonalCollectionChild` at all.
Mentioning this to ask you if this will not introduce any regressions and so we are confident of this change.

All the rest is just refactoring.

Reply:
I see a `isPersonalCollectionChild` check at the `PermissionLink` as it has to be. See #16668

Reply:
Yes, e2e tests failed and told me what to do :)